project/aem/default/etc/aem.yml

```diff
@@ -128,8 +128,10 @@ instance:
 
 # Java options used to launch AEM instances
```

Comment:
@tomasz-sobczyk-wttech @mmajchrzak 
AEMC will provide 11.0.x by default; but could totally easily override it by

- enabling globally installed by env var
- specifying concrete path
- specifying concrete download URL